Create ui-c8y-1021-75-0-add-data-hub-filter-option-to-audit-log-list-9105.md (#3330)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-75-0-add-data-hub-filter-option-to-audit-log-list-9105.md

---
date: ""

> title: add DataHub filter option to audit log list (#9105)

Reviewer: ```suggestion
title: Add filter option for DataHub entries to the audit log list
```

> title: Add filter option for DataHub entries to the audit log list

Reviewer 2: ```suggestion
title: Added filter option for DataHub entries to the audit log list
```

> title: Added filter option for DataHub entries to the audit log list

Reviewer 2: ```suggestion
title: Added filter option for DataHub entries to the audit logs list
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63667
version: 1021.75.0
---

> add DataHub filter option to audit log list (#9105)

Reviewer: ```suggestion
To improve the usability of the audit log list, a new filter option for entries created by DataHub has been added. Previously, the audit log list did not allow filtering of DataHub related entries specifically. This has now been made available.
```

> A new filter option for entries created by DataHub has been added to the audit log list. Previously, the audit log list did not allow filtering of DataHub-related entries specifically.

Reviewer 2: ```suggestion
A new filter option for entries created by DataHub has been added to the audit logs list. Previously, the audit logs list did not allow filtering of DataHub-related entries specifically.
```

> To improve the usability of the audit log list, a new filter option for entries created by DataHub has been added. Previously, the audit log list did not allow filtering of DataHub related entries specifically. This has now been made available.

Reviewer 2: ```suggestion
A new filter option for entries created by DataHub has been added to the audit log list. Previously, the audit log list did not allow filtering of DataHub-related entries specifically.
```
